index.html

```diff
@@ -1308,6 +1325,12 @@ <h2 id="privacy-considerations">
         A user agent SHOULD provide a means for the user to review, update, and reset the
         [=permission=] [=permission/state=] of [=powerful features=] associated with an [=origin=].
       </p>
+      <p>
+        Some [=user agents=] support <dfn data-lt="administrator">administrators</dfn> who can
+        override user choices about permissions, either to block permissions or to allow them on
+        certain sites. Overriding user choices in this way can be a privacy problem, so this
+        specification requires that the [=user agent=] inform users when it happens.
```

Comment:
Out of curiosity, do any browsers in managed environments expose admin set permissions today? 

On my work laptop, I see the following in settings in Chrome:
<img width="321" alt="Screenshot 2023-09-15 at 3 30 56 PM" src="https://github.com/w3c/permissions/assets/67283/93c09c3e-c3ba-40b8-a409-329e01ceb1ff">

And the following in Firefox settings:

<img width="715" alt="Screenshot 2023-09-15 at 3 31 00 PM" src="https://github.com/w3c/permissions/assets/67283/d443c132-dedb-4bfe-ab23-737eca078838">

Reply:
Chrome has a couple that are exposed like 
![calendar.google.com site info panel showing Notifications, Allowed, and "Managed by your organization"](https://github.com/w3c/permissions/assets/83420/145ae377-e0e0-43ce-af80-4cc9f9ce62d1) and 
!["HID devices with usages from usage page F1D0" allowed to chrome-extension://ckcendljdlmgnhghiaomidhiiclmapok/. This setting is managed by your administrator.](https://github.com/w3c/permissions/assets/83420/15b660f5-1991-409b-97ea-9bc490c2e9aa)